lua/harpoon/term.lua

```diff
@@ -45,24 +48,27 @@ local function find_terminal(idx)
     return term_handle
 end
 
-M.gotoTerminal = function(idx)
+M.gotoTerminal = function(idx, create_cmd)
     log.trace("gotoTerminal(): Terminal:", idx)
-    local term_handle = find_terminal(idx)
+    local term_handle = find_terminal(idx, create_cmd)
 
     vim.api.nvim_set_current_buf(term_handle.buf_id)
 end
 
-M.sendCommand = function(idx, cmd, ...)
+M.sendCommand = function(idx, cmd, create_cmd)
```

Comment:
This one had me stumped a bit. The most intuitive would be something like:
```lua
.sendCommand(2, ":e term://ipython", "print('hello world!')")
```
however, this breaks current usage and forces the user to address terminal creation for every command for all users.
So this implementation I am proposing only breaks the variable interpolation part, which still can be used by using a table:
```lua
.sendCommand(2, {"echo %s", "hello world"})
```
I like not breaking user space, but I am unsure how to do that in this case.

Reply:
BTW, I'd like to noe that the interpolation feature makes vim filename expansion and ipython magic commands harder to use. Personally, I'd remove it and let the user do it manually with:
```lua
.sendCommand(2, string.format("echo %s", "hello world"))
```